__docs__/wonder-blocks-form/text-area.argtypes.ts

```diff
@@ -0,0 +1,126 @@
+export default {
```

Comment:
For the most part, Storybook was automatically extracting types and descriptions from the jsdocs for props in the component. Some notes around this:
- I didn't redefine the descriptions here so that the single source of truth can be the prop jsdocs!
- I've only configured types explicitly for the props that had unclear autogenerated types. For example, any union type like `"on" | "off"` would get extracted as `"union"`, which isn't too informative
- I categorized the event related props and the style related props since this component has a long list of props!
- (Might be helpful if we come across this later:) After wrapping the component with `forwardRef`, Storybook was no longer autogenerating docs for all the props. I found that defining the component and types a certain way fixed it: https://github.com/storybookjs/storybook/issues/8881#issuecomment-1098119636